README.md

```diff
@@ -138,17 +124,16 @@ update-contributors           Regenerates the contributors html/list
 <br />
 
 ## Usage
-Here's the [getting started](https://getbux.io) with BUX
 
 ```javascript
 import bsv from 'bsv';
-import { BuxClient } from 'js-buxclient';
+import { SpvWalletClient } from 'spv-wallet-js-client';
 
 const server = "http://localhost:3003/v1";
 const transportType = 'http'; // or graphql
 const xPriv = bsv.HDPrivateKey.fromRandom();
```

Comment:
we should suggest generating keys from this lib, not from bsv lib.

Reply:
Changed